Composer/packages/client/src/App.tsx

```diff
@@ -71,6 +72,14 @@ const topLinks = (botLoaded: boolean) => {
       exact: false,
       underTest: true, // will delete
     },
+    {
+      to: '/notifications',
+      iconName: 'Warning',
+      labelName: formatMessage('Notifications'),
+      activeIfUrlContains: '/notifications',
```

Comment:
where did this `activeIfUrlContains` convention come from?

Reply:
I found that the activeIfUrlContains was not useful for the component. I have removed it now.